x/hash/hash.go

```diff
@@ -22,7 +22,7 @@ func BytesHash(d []byte) Hash {
 
 // StringHash returns the hash of a string.
 func StringHash(s string) Hash {
-	return BytesHash(unsafe.ToBytes(s))
+	return BytesHash(safe.ToBytes(s))
```

Comment:
Given the usage here is confined with a single line within a single function, and the converted bytes are not used outside the function at all, it should be fine to use `unsafe` in this simple context.